### PR TITLE
Use warning when id < 0

### PR DESCRIPTION
**Pull request by workingloong**
The id < 0 may be used as a default value, so we should raise a warning not an error.

### CODE REVIEW

File: elasticdl/python/elasticdl/layers/embedding.py [hunk 1]
```diff
@@ -140,7 +141,7 @@ def _check_id_valid(self, ids):
                 % (first_may_exceed_id, self.input_dim)
             )
         if any(ids < 0):
-            raise ValueError(
+            logger.warning(
```
Comment (chunyang-wen) [inline elasticdl/python/elasticdl/layers/embedding.py]: It's better that we can make it configurable.

Comment (workingloong) [reply elasticdl/python/elasticdl/layers/embedding.py]: But, how and where to config it? `tf.nn.safe_embedding_lookup_sparse` auto prunes invalid IDs ( < 0) and `tf.nn.embedding_lookup_sparse` will raise an error with  invalid IDs ( < 0).

File: elasticdl/python/elasticdl/layers/embedding.py [hunk 2]
```diff
@@ -140,7 +141,7 @@ def _check_id_valid(self, ids):
...
                 "The embedding id cannot be less than zero. "
```
Comment (terrytangyuan) [inline elasticdl/python/elasticdl/layers/embedding.py]: Here it says cannot be zero. Need to modify the message to tell users what could go wrong if id < 0

**Comment by workingloong** [comment]
I will refactor the embedding implementation in embedding_delegate.py and prune the invalid IDs (<0) like `tf.nn.safe_embedding_lookup_sparse`. So I close the PR.

**Pull request closed without merging**